spec/factories/access_token.rb

```diff
@@ -6,7 +6,7 @@
     uid { '12342' }
     info do
       {
-        name: FFaker::Name.name,
+        name: "is-#{FFaker::Name.name[0, 27]}",
```

Comment:
Why "is-" needed here?

Reply:
The user name have to have at least 2 symbols. The 'is' was created to guarantee it. _Instead to check a FFaker result._

Reply:
Why not to generate two words explicitly? I mean write exactly what you need without this cryptic?